features/prompt-registry/snippets.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Snippets"
+icon: "link"
+---
+
+Snippets in PromptLayer allow you to modularize and reuse pieces of your prompt templates, much like using building blocks to create a larger structure. This feature enables you to compose complex templates by referencing other prompt template versions within a parent template.
+
+## How Snippets Work
+
+A snippet is a reference to a prompt template that can be inserted into another template. You can reference templates in three ways:
+
+1. **By Template Name**: `@@@template_name@@@` - This will automatically use the latest version of the referenced template.
+2. **By Template Version Number**: `@@@template_name@version_number:{number}@@@` - This points to a specific version of the template.
+3. **By Template Label**: `@@@template_name@label:{label_name}@@@` - This uses the version of the template that has the specified label.
+
+The system is designed to encourage iterative development, allowing you to test and refine smaller components of your prompts before integrating them into larger templates.
```

Comment:
remove "The system is designed to encourage iterative development, allowing you to test and refine smaller components of your prompts before integrating them into larger templates."